pkg/reconciler/pipelinerun/resources/apply_test.go

```diff
@@ -2056,6 +2056,77 @@ func TestApplyWorkspaces(t *testing.T) {
 	}
 }
 
+func TestApplyFinallyResultsToPipelineResults(t *testing.T) {
```

Comment:
Do we also need a test case for object results?

Also, it would be good to have a pipelinerun reconciler test for this functionality. It looks like these test cases ensure that the syntax "finally.<taskname>.results.<resultname>" can be used to apply results, but we also want to make sure that taskrun results can be collected from the pipeline's finally tasks and then applied to pipeline results.

Reply:
Yes, I will write a test case for object results in order to make my testing more extensive. As for your second comment, I will create the test accordingly, will ping you if need be as to how to create the test.

Reply:
Added test case for object result

Reply:
added test case to reconciler.